libs/application/templates/new-primary-school/src/lib/messages.ts

```diff
@@ -742,8 +742,8 @@ export const newPrimarySchoolMessages: MessageDir = {
       description:
         'This refers to diagnoses such as epilepsy, diabetes, or blood clotting disorders',
     },
-    requestMedicationAssistance: {
-      id: 'nps.application:different.needs.allergies.and.intolerances.request.medication.assistance',
+    requestsMedicationAdministration: {
+      id: 'nps.application:different.needs.allergies.and.intolerances.request.medication.administration',
```

Comment:
Shouldn't 'request' in id be plural? (requests)
```suggestion
    requestsMedicationAdministration: {
      id: 'nps.application:different.needs.allergies.and.intolerances.requests.medication.administration',
```

Reply:
Fixed